recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ test:
     - rav1e --help
     - test -f ${PREFIX}/lib/librav1e${SHLIB_EXT}             # [unix]
     - if not exist %LIBRARY_PREFIX%\\bin\\rav1e.dll exit 1   # [win]
+    - if not exist %LIBRARY_PREFIX%\\lib\\rav1e.lib exit 1   # [win]
```

Comment:
```suggestion
```
Instead of changing the name of this file on Windows, can we first try adding the PC file to the list of files which contain the host prefix. Then copying [this cmake finding script](https://github.com/AOMediaCodec/libavif/blob/262e11d9f4018ee89aacd153141f444b6db025a2/cmake/Modules/Findrav1e.cmake) into the recipe and just testing if CMake can locate the library?